discretisedfield/mesh.py

```diff
@@ -229,6 +233,77 @@ def __init__(
         if "isplane" not in attributes.keys():
             self.attributes["isplane"] = False
 
+    @property
+    def bc(self):
+        """Boundary condition for the mesh.
+
+        Periodic boundary conditions can be specified by passing a string containing one
+        or more characters from ``self.region.dims`` (e.g. ``'x'``, ``'yz'``, ``'xyz'``)
+        . Neumann or Dirichlet boundary conditions are defined by passing ``'neumann'``
+        or ``'dirichlet'`` string. Neumann and Dirichlett boundary conditions are still
+        experimental.
+
+        Returns
+        -------
+        str
+
+            A string representing periodic boundary condition along one or more axes, or
+            Dirichlet or Neumann boundary condition. The string is empty if no boundary
+            condition is defined.
+        """
+        return self._bc
+
+    @bc.setter
+    def bc(self, bc):
+        if not isinstance(bc, str):
+            raise TypeError("Value of bc must be a string.")
+        bc = bc.lower()
+        if bc not in {"neumann", "dirichlet", ""}:
+            for char in bc:
+                if char not in self.region.dims:
+                    raise ValueError(f"Axis {char} is absent in {self.region.dims}.")
+                elif bc.count(char) > 1:
+                    raise ValueError(f"Axis {char} is present more than once.")
+
+        self._bc = bc
+
+    @property
+    def cell(self):
+        """The cell dimensions of the mesh.
```

Comment:
Would `cell size` be better?
```suggestion
        """The cell size of the mesh.
```